website_oca_psc_team/demo/assets_demo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+Copyright 2020 Tecnativa - Pedro M. Baeza
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+    <template id="assets_psc_team_frontend_demo" inherit_id="web.assets_frontend">
+        <xpath expr="//script[last()]" position="after">
+            <script
+                type="text/javascript"
+                src="/website_oca_psc_team/static/src/js/psc_team_tour.js"
+            />
+        </xpath>
+    </template>
```

Comment:
Use `website.assets_tests` instead of adding to demo files. 

So delete this file